src/registrar/tests/test_transition_domain_migrations.py

```diff
@@ -19,6 +19,233 @@
 from .common import less_console_noise
 
 
+class TestOrganizationMigration(TestCase):
+    def setUp(self):
+        """ """
+        # self.load_transition_domain_script = "load_transition_domain",
+        # self.transfer_script = "transfer_transition_domains_to_domains",
+        # self.master_script = "load_transition_domain",
+
+        self.test_data_file_location = "registrar/tests/data"
+        self.test_domain_contact_filename = "test_domain_contacts.txt"
+        self.test_contact_filename = "test_contacts.txt"
+        self.test_domain_status_filename = "test_domain_statuses.txt"
+
+        # Files for parsing additional TransitionDomain data
+        self.test_agency_adhoc_filename = "test_agency_adhoc.txt"
+        self.test_authority_adhoc_filename = "test_authority_adhoc.txt"
+        self.test_domain_additional = "test_domain_additional.txt"
+        self.test_domain_types_adhoc = "test_domain_types_adhoc.txt"
+        self.test_escrow_domains_daily = "test_escrow_domains_daily"
+        self.test_organization_adhoc = "test_organization_adhoc.txt"
+        self.migration_json_filename = "test_migrationFilepaths.json"
+
+    def tearDown(self):
+        # Delete domain information
+        Domain.objects.all().delete()
+        DomainInformation.objects.all().delete()
+        DomainInvitation.objects.all().delete()
+        TransitionDomain.objects.all().delete()
+
+        # Delete users
+        User.objects.all().delete()
+        UserDomainRole.objects.all().delete()
+
+    def run_load_domains(self):
+        # noqa here because splitting this up makes it confusing.
+        # ES501
+        with patch(
+            "registrar.management.commands.utility.terminal_helper.TerminalHelper.query_yes_no_exit",  # noqa
+            return_value=True,
+        ):
+            call_command(
+                "load_transition_domain",
+                self.migration_json_filename,
+                directory=self.test_data_file_location,
+            )
+
+    def run_transfer_domains(self):
+        call_command("transfer_transition_domains_to_domains")
```

Comment:
**comment/non-blocking** honestly this command call coveys its purpose more  then just "run transfer_domains". Some meaning gets lost here with hiding it in the function whose name doesn't describe the purpose as well. Comment though would solve this